GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1311,7 +1321,24 @@ private void ApplyPatchToolStripMenuItemClick(object sender, EventArgs e)
 
         private void GitBashToolStripMenuItemClick1(object sender, EventArgs e)
         {
-            Module.RunBash();
+            if (!GitBash.DropDownButtonPressed)
+            {
+                Module.RunBash();
```

Comment:
This is different from `RunShell` if you don't have `bash.exe` on your path.

Reply:
I've decided to remove the duplicate one in the menu which has the advantage to solve this issue...